src/simulation/GameCell.ts

```diff
@@ -137,6 +143,12 @@ export class Pop {
   }
 }
 
+export interface IPopCoordinates {
```

Comment:
What is this? Also, use `x` not `xCoord`

Reply:
It's a wrapper around the pop class to allow it to contain coordinate data, is deprecated